sdk/communication/azure-communication-callingserver/azure/communication/callingserver/_call_connection.py

```diff
@@ -28,19 +37,42 @@
 
 class CallConnection(object):
     def __init__(
-            self,
-            call_connection_id,  # type: str
-            call_connection_client  # type: CallConnectionsOperations
-        ): # type: (...) -> None
+        self,
+        call_connection_id,  # type: str
+        call_connection_client  # type: CallConnectionsOperations
+    ): # type: (...) -> None
 
         self.call_connection_id = call_connection_id
         self._call_connection_client = call_connection_client
 
+    @distributed_trace()
+    def create_call(
```

Comment:
why we need this api? the is a create_call_connection() api in _callingserver_client.py already. I think this is a duplicate one.